apps/dash-image-segmentation/app.py

```diff
@@ -515,15 +640,27 @@ def annotation_react(
 
 # ======= Callback for modal popup =======
```

Comment:
```suggestion
# ======= Callback for modal popup =======
```
This is nitpicky (not a big deal haha) but maybe the comments could be normalized to all have the same style, e.g., all have === * === or not etc.

Reply:
I like that! Thanks for the suggestion! I think all other comments are just plain old `#`. So I'd say we take this one's fancy-style away. 
Did you change anything in your suggestion? I can't see the difference.

Reply:
No just wanted to point out the comment style